src/Traits/Hooks/UseDashboardZoneOne.php

```diff
@@ -130,8 +133,11 @@ protected function loadPsAccounts(): string
      * 
      * @return bool
      */
-    private function ensurePsAccountIsEnabled(): bool {
-        $accountsInstaller = $this->get('mbo.ps_accounts.installer');
+    private function ensurePsAccountIsEnabled(): bool
+    {
+        if (version_compare(_PS_VERSION_, "9.0.0", ">=")) return false;
```

Comment:
Don't forget to remove this after publication of v9.0.0 on addons and/or official releases for the module